DataStandardization/create_data_to_process_files.py

```diff
@@ -5,19 +5,37 @@
 
 RelevantTypes = ()
```

Comment:
Per the [PEP 8 rules](https://www.python.org/dev/peps/pep-0008/#prescriptive-naming-conventions), this probably should be `relevant_types` or `RELEVANT_TYPES`.